src/doc/unstable-book/src/the-unstable-book.md

```diff
@@ -5,16 +5,16 @@ each one organized by a "feature flag." That is, when using an unstable
 feature of Rust, you must use a flag, like this:
 
 ```rust
-#![feature(box_syntax)]
+#![feature(inline_const)]
```

Comment:
nit: this still probably stabilize soon, maybe another feature would be better

Maybe `generators` would be a good one, that's far off and probably won't change too much?